### PR TITLE
Release notes for v5.5

### PR DESCRIPTION
**Pull request by ibrand**
release notes 🚀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ Spoke is an open source text-distribution tool for organizations to mobilize sup
 
 Spoke was created by Saikat Chakrabarti and Sheena Pakanati, and is now maintained by MoveOn.org.
 
-The latest version is [5.4](https://github.com/MoveOnOrg/Spoke/tree/v5.4) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/main/docs/RELEASE_NOTES.md#v54))
+The latest version is [5.4](https://github.com/MoveOnOrg/Spoke/tree/v5.5) (see [release notes](https://github.com/MoveOnOrg/Spoke/blob/main/docs/RELEASE_NOTES.md#v55))
```
Comment (schuyler1d) [inline README.md]: the link text needs to be 5.5

File: docs/RELEASE_NOTES.md [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+    - We put a lot of work into this interface to accomodate upcoming features, radically improve the mobile (and generally cross-screen support) and address some issues that regularly come up for texters.
+    - Please send your experience reports in testing
+    - More context can be seen at [RFC: New Texter UI](https://github.com/MoveOnOrg/Spoke/pull/1522)
+  - EXPERIMENTAL_TAGS: Tagging users instead of just saving question responses is a very common request, and we have a great volunteer team developing these features. This first step is creating an admin interface to create the tags. There will be more to come, but you can preview and test this development by enabling the environment variable. Thanks to @aschneit and @filafb and TKTK
+
+(a cypress stub was also merged into main for e2e tests recently)
+
+Thanks to all the contributors part of this release including:
+[aschneit](https://github.com/aschneit),
```
Comment (schuyler1d) [inline docs/RELEASE_NOTES.md:16]: maybe alphabetize?

**Review by schuyler1d — DISMISSED** [review]
approving, but do update the link text to say 5.5 first.

File: docs/RELEASE_NOTES.md (hunk 2 above)
Comment (ibrand) [reply docs/RELEASE_NOTES.md:16]: Oh I'm down with that! It's basically always been random. Nice to have a system

File: README.md (hunk 1 above)
Comment (ibrand) [reply README.md]: Wow. This is such a blindspot of mine. Thank you!

**Review by schuyler1d — APPROVED** [review]
:fireworks:

**Pull request merged**